README.rst

```diff
@@ -1,12 +1,33 @@
 
-*******
-Cantera
-*******
+.. Cantera
 
-Version 2.4.0b1 (development)
+.. image:: http://cantera.github.io/docs/sphinx/html/_static/cantera-logo.png
+   :target: http://cantera.github.io/docs/sphinx/html/index.html#
+   :alt: cantera logo
+   :width: 675px
+   :align: center
+   
 
-.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.170284.svg
+.. |doi| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.170284.svg
    :target: https://doi.org/10.5281/zenodo.1174508
+   
+.. |codecov| image:: https://img.shields.io/codecov/c/github/Cantera/cantera/master.svg
+   :target: https://codecov.io/gh/Cantera/cantera?branch=master
```

Comment:
I'm finding these `shields.io` badges take an exceedingly long time to load and frequently time out. They're cute, but do they really add anything?

Reply:
Some of them are not absolutely critical. I personally find them useful on other repos for making quick overviews of the state of a github project.

Reply:
The Github commits badge can be removed as it does not convey a very rich project metric.